File: python/test_serial2.js
var serialport = require('serialport');
var SerialPort = serialport.SerialPort;

var serialPort = new SerialPort('/dev/cu.usbmodem1411', {
  baudrate: 9600
});

serialPort.on('open', function () {
  console.log('open');
  serialPort.on('data', function(data) {
    console.log(data);
  });
});

File: python/test_serial2.py
import serial
import time

ser = serial.Serial('/dev/tty.SLAB_USBtoUART8', 9600, timeout = .1)
# ser = serial.Serial('/dev/tty.usbmodem1411', 9600, timeout = 1)

print ("Starting up")
connected = False
commandToSend = "1324\r\n" # get the distance in mm

while True:
    print("ARDUINO")
    print("\n[KIRIM]===================================")
    print ("Writing: ",  bytes(commandToSend, 'UTF-8'))
    ser.write(bytes(commandToSend, 'UTF-8'))
    while True:
        try:
            print("\n[TERIMA]==================================")
            readOut = ser.readline()
            print ("Reading: ", readOut) 
            break
        except:
            pass
    print ("Restart")
    ser.flush() #flush the buffer